### PR TITLE
Add color to the Character commands

### PR DESCRIPTION
**Pull request by lewisvive**
Added ability to set print color.

### CODE REVIEW

File: ESCPOS_NET/Emitters/Enums/Color.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace ESCPOS_NET.Emitters
+{
+    [Flags]
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.SpacingRules", "SA1025:Code should not contain multiple whitespace in a row", Justification = "Enums are easier to read if they have whitespace alignment.")]
```
Comment (lukevp) [inline ESCPOS_NET/Emitters/Enums/Color.cs:6]: I agree with the justification, but the enum below doesn't have whitespace alignment. Can you move this to the suppressions file instead?

**Review by lukevp — CHANGES_REQUESTED** [review]
Thanks for the PR! minor change on suppressions and there's a conflict that must be resolved before merge.

**Comment by FedericoBernardi** [comment]
Any plan to merge it in master?

**Review by lukevp — APPROVED** [review]

**Comment by lukevp** [comment]
Hi @FedericoBernardi  I was waiting for resolution of this comment: 
image

However, sinc eyou asked for this I will go ahead and merge and resolve this so we can move forward with this functionality and I'll make the changes directly.

**Pull request merged**